potodo/_github.py

```diff
@@ -24,9 +25,13 @@ def get_repo_url(repo_path: str) -> str:
                 stderr=subprocess.STDOUT,
             )
         except subprocess.CalledProcessError as e:
+            logging.fatal(
+                f'Unknown error. `{" ".join(e.cmd)}` returned "{e.output.rstrip()}".'
+            )
```

Comment:
I'm not sure it's usefull to log here, as the runtime error is not handled, so it'll crash the program with the message anyway.

Also better use `logging.exception` so it'll automatically catch the exception (test it using `git remote rename origin o; git remote rename upstream u` :P)